lib/cc/engine/analyzable_files.rb

```diff
@@ -27,13 +26,17 @@ def fetch_files(paths)
       end
 
       def build_files_with_inclusions(inclusions)
-        fetch_files(inclusions)
+        if inclusions == ["./"]
```

Comment:
That example would also cause similar problems with the current implementation. Thoughts on stripping leading `./` instead?